pkg/metaserver/spd/manager.go

```diff
@@ -106,6 +117,50 @@ type serviceProfilingManager struct {
 	fetcher SPDFetcher
 }
 
+func (m *serviceProfilingManager) ServiceExtendedIndicator(ctx context.Context, pod *v1.Pod, indicators interface{}) (bool, error) {
```

Comment:
can we return indicator object as well? because we might need to fetch configs from extended indicator.